lib/ifconfig/sunos/ifconfig.rb

```diff
@@ -31,7 +31,8 @@ def initialize(input=nil,netstat=nil,verbose=nil)
         when /\s+ether\s+/im
           @ifaces[iface_name] = EthernetAdapter.new(iface_name,iface,netstat)
         else
-          puts "Unknown Adapter Type: #{iface}" if @verbose
+          # fallback for virtual nics like bge:0
+          @ifaces[iface_name] = EthernetAdapter.new(iface_name,iface,netstat)
```

Comment:
Well, I think you are right. It should be ```@ifaces[iface_name] = NetworkAdapter.new(iface_name,iface,netstat)``` for the fallback. Yes, I think virtual nics, or vnics are called alias in linux, but I am not into linux that much. Would you pull the request if I change the line?